<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPackagesDialog</class>
 <widget class="QDialog" name="EditPackagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Synchronized Packages</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,1,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Extra Filters</string>
     </property>
     <layout class="QVBoxLayout" name="checkLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Enter a regular expression to filter the lists below</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Installed Packages:</string>
         </property>
         <property name="buddy">
          <cstring>localPackageListView</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="localPackageListView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="toolTip">
          <string>Add packages to sync list (Ctrl+Right)</string>
         </property>
         <property name="text">
          <string>-&gt;</string>
         </property>
         <property name="icon">
          <iconset theme="go-next">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Right</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeButton">
         <property name="toolTip">
          <string>Remove packages from sync list (Ctrl+Left)</string>
         </property>
         <property name="text">
          <string>&lt;-</string>
         </property>
         <property name="icon">
          <iconset theme="go-previous">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Left</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="clearAllButton">
         <property name="toolTip">
          <string>Clear all selections (Esc)</string>
         </property>
         <property name="text">
          <string>X</string>
         </property>
         <property name="icon">
          <iconset theme="edit-clear-all-symbolic">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="shortcut">
          <string>Esc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Synchronized Packages:</string>
         </property>
         <property name="buddy">
          <cstring>dbPackageListView</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="dbPackageListView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>localPackageListView</tabstop>
  <tabstop>dbPackageListView</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>clearAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditPackagesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditPackagesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
